focus_converter_base/focus_converter/conversion_configs/aws/effective_cost_S001.yaml

```diff
@@ -0,0 +1,17 @@
+plan_name: conversion plan for EffectiveCost
+conversion_type: sql_query
+conversion_args: >
+    SELECT
+    *,
```

Comment:
Generally speaking, why do you need to `SELECT *`?

does it work when you just do `SELECT CASE .... END AS name`?

Reply:
select * is needed so that it does not drop any columns for the next steps to work.